### DISCUSSION TITLE
Policy.rates for negative activities

### DISCUSSION THREAD

**Discussion by zanebclark** [openmobilityfoundation/curb-data-specification#59] [Data Specification]
If a [Rule](https://github.com/openmobilityfoundation/curb-data-specification/tree/main/curbs#rule) has a negative activity (e.g. `no parking`), what purpose would the `Rule.rate` array serve? Can we: 

1) Clarify the purpose of a negative activity `Rule.rate` or
2) Document that a negative activity `Rule.rate` array should be empty

**Comment by schnuerle** [comment]
Good idea. Take a look at updates to the Rate.rule [description](https://github.com/openmobilityfoundation/curb-data-specification/blob/feature-release-work-1/curbs/README.md#rule), and the new sentence at the end of the [Activities](https://github.com/openmobilityfoundation/curb-data-specification/blob/feature-release-work-1/curbs/README.md#activities) description.
